lib/dynflow/config.rb

```diff
@@ -85,6 +85,14 @@ def method_missing(name)
       true
     end
 
+    config_attr :scheduler_options, Hash do
+      { :poll_interval => 15, :time_source => -> { Time.now.utc } }
+    end
+
+    config_attr :scheduler_class, Class, NilClass do
+      Schedulers::Polling
```

Comment:
why not having just one config option: `:scheduler` and expecting it to provide the scheduler object itself?